OpenLaTeXStudio/EditorModule/src/latexstudio/editor/EditorTopComponent.java

```diff
@@ -326,4 +336,35 @@ private CompletionProvider createCompletionProvider() {
 
         return provider;
     }
+
+    private void displayCloudStatus() {
+
+        boolean isConnected = false;
+        String message;
+        DbxAccountInfo info = null;
+
+        // Check Dropbox connection
+        DbxClient client = DbxUtil.getDbxClient();
+        if (client != null) {
+            String userToken = client.getAccessToken();
+            if (userToken != null && !userToken.isEmpty()) {
+                try {
+                    info = client.getAccountInfo();
+                    isConnected = true;
+                } catch (DbxException ex) {
+                    Exceptions.printStackTrace(ex);
```

Comment:
We shouldn't print the stacktrace in this case. If I have my application conected with Dropbox and I start the application. but I'm not conected to the Internet, I get the stacktrace in a pop up window right at thr startup which is not the best.